jekyll/_cci2/contexts.md

```diff
@@ -21,13 +21,15 @@ Contexts are created on the Settings page of the CircleCI application, in the Or
 
 To use environment variables set on the Contexts page, the person running the workflow must be a member of the organization for which the context is set and the rule must allow access to all projects in the org. The default context name is `org-global` and the default rule allows access to all projects in the org.
 
+Context names must be unique for each Github or Bitbucket organization. For CircleCI installations on your own server or private cloud, Context names must be unique across the installation.
+
 ## Using a Context
 
-1. Navigate to the Settings > Contexts page in the CircleCI application. The Create a Context button appears.
+1. Navigate to the Settings > Contexts page in the CircleCI application. The Create a Context button appears if you have no existing contexts.
 
-2. Click the Create Contexts button. The default name and rules appear with an Add a Resource button.
+2. Click the Create Contexts button. The default name and rules appear with an Add Environment Variable button.
 
-3. Click the Add a Resource button and fill in the variable name and value. Click the Add Variable button to save it.
+3. Click the Add Environment Variable button and type in the variable name and value. Click the Add Variable button to save it.
```

Comment:
I'm not sure if `type` is misleading here as you can copy paste that value as well (which is actually more recommended for sensitive values)

Reply:
thanks, changed to copy/paste